andela/travel_tool_front
# #162305548 Modify date range pickers on the dashboard

#### What does this PR do?
- This PR modifies travel calendar date range picker to match analytics' date range picker

#### Description of Task to be completed?
- This PR modifies travel calendar date range picker to match analytics' date range picker

#### How should this be manually tested?
- Clone the repo `https://github.com/andela/travel_tool_front.git`
- Checkout to the branch:
```ft-date-range-filter-161840270```
- Install dependencies:
```yarn install```
`yarn test`
- All the tests should be passing
- Start the application:
```yarn start```

- Login to the application
- Navigate to the dashboard page on the sidebar
-  Click on the date button on the dashboard header
- The scroll down to the travel calendar section and click the date button. 
- You should be able to get results as shown in the screenshot below.

#### Any background context you want to provide?
1. To use the Dashboard - Create your standalone component and plug it into the dashboard component.
2. You should have the user role of a Travel Administrator or Super Administrator to be able to see the `manage` page. To do that:
  - Download `Postico` and connect to your travela database with relevant details: Db name, password, port, user
  - Click `userRoles` table and set the `roleId` to that of a Travel Administrator(29187) or Super[]Administrator(10948) 

#### What are the relevant pivotal tracker stories?
[#162305548](https://www.pivotaltracker.com/story/show/162305548)

#### Screenshots (if appropriate)
##### Analytics Section
<img width="1440" alt="screenshot 2018-12-11 at 09 48 41" src="https://user-images.githubusercontent.com/22371596/49783253-49002480-fd2a-11e8-9bd5-4f9c4c5c665e.png">

##### Travel Calendar section
<img width="1440" alt="screenshot 2018-12-11 at 09 49 12" src="https://user-images.githubusercontent.com/22371596/49783297-6b923d80-fd2a-11e8-876d-d18ce3ab1d62.png">
